source/styles.brs

```diff
@@ -1,78 +1,17 @@
-function GetStyles() as Object
-    return {
-        classes: {
-            ' HomeScene
-            homeList: {
-                itemComponentName: "HomeListItem",
-                numRows:"3",
-                translation: "[50, 150]",
-                itemSize: "[ 2000, 350 ]",
-                showRowLabel: "[ true ]",
-                rowLabelOffset: "[ [0, 15], [0, 15], [0, 15] ]",
-                drawFocusFeedback:true,
-                vertFocusAnimationStyle:"fixedFocusWrap",
-                rowFocusAnimationStyle: "fixedFocusWrap",
-                visible: true,
-                rowItemSize:"[[500, 285]]",
-                itemSpacing:"[0, 80]",
-                rowItemSpacing: " [[30, 0]]"
-            },
-            video: {
-                   width:"1920",
-                   height:"1080",
-                   visible:false
-            },
+function getClassesFromJson () as object
+      info = ReadAsciiFile("pkg:/source/styles.json")
```

Comment:
there should be a task set up to read external files (even if they are in the filesystem).
⚠️ Note that this function is being called from three different places, so, you are loading and parsing the SAME json file three times...

I suggest to create a promise and follow the same pattern as you did for loading movies